doc-changelog/action.yml

```diff
@@ -136,6 +135,7 @@ runs:
         PR_BRANCH: ${{ github.event.pull_request.head.ref }}
         PR_TITLE: "${{ github.event.pull_request.title }}"
         PR_NUMBER: ${{ github.event.number }}
+        GH_TOKEN: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
```

Comment:
Composite actions can't use secrets. You need to pass them via input (i.e. token). So, in your case, we need to add to the description the fact that the token needs the contents: write and pull-requests: write permissions + we encourage users to make use of the ``PYANSYS_CI_BOT_TOKEN `` for organization purposes.

Reply:
But I think you fixed that already 😄 good job!